core/src/main/java/hudson/util/CompressedFile.java

```diff
@@ -115,7 +116,7 @@ public String loadAsString() throws IOException {
         StringBuilder str = new StringBuilder((int)sizeGuess);
 
         try (InputStream is = read();
-             Reader r = new InputStreamReader(is)) {
+             Reader r = new InputStreamReader(is, Charset.defaultCharset())) {
```

Comment:
Maybe? No idea what this class is even used for.

Reply:
It's unused. I deprecated it and will remove in a few months if no callers show up.